test-utils/utils.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import {render} from '@testing-library/react';
+import type {RenderOptions} from '@testing-library/react';
+import {memoize} from 'lodash';
+
+import type {ThemeContextProps} from '../src';
+import {ThemeContext} from '../src/components/theme/ThemeContext';
+
+const themeContextValue: ThemeContextProps = {
+    direction: 'ltr',
+    theme: 'light',
+    themeValue: 'light',
+};
+
+function Providers({children}: {children: React.ReactElement}) {
+    return <ThemeContext.Provider value={themeContextValue}>{children}</ThemeContext.Provider>;
+}
+
+const createWrapper = memoize(
```

Comment:
Memoization is really necessary?

Reply:
Taking a closer look, memoization is unnecessary here.